### ISSUE TITLE
bug: enters invalid state when used inside help buffer

### ISSUE THREAD

**Issue by ViRu-ThE-ViRuS** [karb94/neoscroll.nvim#37]
**Neoscroll config:**
``` lua
require('neoscroll').setup({ })

local mappings = {}
mappings['<C-u>'] = {'scroll', {'-vim.wo.scroll', 'true', '50'}}
mappings['<C-d>'] = {'scroll', {'vim.wo.scroll', 'true', '50'}}
mappings['<C-y>'] = {'scroll', {'-0.10', 'false', '50'}}
mappings['<C-e>'] = {'scroll', {'0.10', 'false', '50'}}
mappings['{'] = {'scroll', {'-0.10', 'true', '50', nil}}
mappings['}'] = {'scroll', {'0.10', 'true', '50', nil}}
mappings['zt'] = {'zt', {'50'}}
mappings['zz'] = {'zz', {'50'}}
mappings['zb'] = {'zb', {'50'}}

require('neoscroll.config').set_mappings(mappings)
```

**Bug:** with the given config, when using any of the mappings inside a help buffer, for example the one which opens with the command `:help lua`, neoscroll outputs the following error message: 

```
E5108: Error executing lua .../pack/packer/start/neoscroll.nvim/lua/neoscroll/init.lua:198: Vim(normal):E471: Argument required: normal! 
```

**Side Effect:** cursor becomes invisible after the above mentioned error message in all windows and buffers. neoscroll mappings no longer respond.

**Comment by karb94** [comment]
Was this fixed with the last PR or is this still an issue? Haven't been able to test it yet.

**Comment by ViRu-ThE-ViRuS** [comment]
yeah! it was fixed one hour ago by #36
